packages/core/package.json

```diff
@@ -10,22 +10,22 @@
       "import": "./dist/index.js"
     },
     "./lib/predefined_bootstrap_nodes": {
-      "types": "./dist/lib/predefined_bootstrap_nodes.d.ts",
```

Comment:
Sorry, I meant remove the `lib` folder from `packages/proto`.

Yes, it would make sense to also remvoe `lib` folder from `packages/core` but I would do that in a separate PR because it's going to be cumbersome to find all the places that needs changes. I'd limit the scope of this PR to extract the `proto` package. Follow-up PR can handle the cleaning of `core` package.